depends/install_extra_test_images.sh

```diff
@@ -1,15 +1,12 @@
-#!/bin/bash
+#!/usr/bin/env bash
 # install extra test images
 
-# Use SVN to just fetch a single Git subdirectory
-svn_export()
-{
-    if [ ! -z $1 ]; then
-        echo ""
-        echo "Retrying svn export..."
-        echo ""
-    fi
-
-    svn export --force https://github.com/python-pillow/pillow-depends/trunk/test_images ../Tests/images
-}
-svn_export || svn_export retry || svn_export retry || svn_export retry
+archive=test-images-main
+
+./download-and-extract.sh $archive https://github.com/python-pillow/test-images/archive/refs/heads/main.tar.gz
```

Comment:
```suggestion
./download-and-extract.sh $archive https://github.com/python-pillow/test-images/archive/main.tar.gz
```
Copying the simpler URL you used in test-windows.yml